config/locales/api/en.yml

```diff
@@ -470,4 +472,18 @@ en:
     oral_cancer: "Oral cancer"
     breast_cancer: "Breast cancer"
     cervical_cancer: "Cervical cancer"
-
+    # Ethiopia
+    et_total_bp: "Total BP screening done"
+    et_total_bs: "Total blood sugar screening done"
```

Comment:
Might have to change this to `Total BS screening done` if design team agrees to [this comment](https://www.figma.com/file/vmZlmlFSCOrEYzTMkqsNB8?node-id=29876:50212#403255266)